mgmtsystem_action/__openerp__.py

```diff
@@ -20,31 +20,25 @@
 ##############################################################################
 {
     "name": "Management System - Action",
-    "version": "8.0.1.2.0",
+    "version": "9.0.1.0.0",
     "author": "Savoir-faire Linux,Odoo Community Association (OCA)",
     "website": "http://www.savoirfairelinux.com",
     "license": "AGPL-3",
     "category": "Management System",
-    "description": """\
-This module enables you to manage the different actions of your management
-system:
-  * immediate actions
-  * corrective actions
-  * preventive actions
-  * improvement opportunities.
-""",
-    "depends": ['mgmtsystem', 'crm_claim'],
+    "depends": ['mgmtsystem', 'crm_claim', 'mail'],
```

Comment:
I can see that Actions inherit from `crm.claim`, so the dependency is correct.
But this dependency is an implementation detail, and not functionally needed.
It comes from v6, where it was interesting to inherit interesting features from `crm.claim` to avoid repeated code, such as messaging. Other core models, such as `project.issue` did the same.

However, since v7 the core modules stopped doing this, and many inetrcation features were moved into the `mail` module.

IMO it's about time we drop the dependency, and implement directly on Actions the features we might need from `crm.claim`. 

Can you check if this means a significant effort?